cuda/common/include/pcl/cuda/common/eigen.h

```diff
@@ -102,7 +101,8 @@ namespace pcl
   
     inline __host__ __device__ bool isMuchSmallerThan (float x, float y)
     {
-      float prec_sqr = FLT_EPSILON * FLT_EPSILON; // copied from <eigen>/include/Eigen/src/Core/NumTraits.h
+      float prec_sqr =
```

Comment:
Comment deleted because I can't find such statement in `<eigen>/include/Eigen/src/Core/NumTraits.h`.

Reply:
Seems like that was moved to [MathFunctions.h](https://gitlab.com/libeigen/eigen/-/blob/master/Eigen/src/Core/MathFunctions.h). Maybe change the comment to `inspired by Eigen's implementation` or similar?